docs/source/roles.md

```diff
@@ -6,22 +6,29 @@ A TRE conforming to the SATRE specification should provide a broadly similar exp
 There is not necessarily a one-to-one mapping between roles and people.
 One person can have multiple roles.
 
-## TRE users
+(user_roles)=
 
-The researchers working on projects that involve logging into a TRE to access data.
+## TRE user roles
+
+Roles for non-administrators working on projects that involve logging into a TRE to access data.
```

Comment:
```suggestion
Roles for people using the TRE to conduct research or analyse data in the TRE.
```
I think it's better to say what the role is, rather than what it isn't.